pkgs/applications/networking/qv2ray/default.nix

```diff
@@ -74,9 +74,11 @@ mkDerivation rec {
 
   meta = with lib; {
     description = "An GUI frontend to v2ray";
-    homepage = "https://qv2ray.net";
```

Comment:
The official website is no longer accessible. qv2ray has almost stopped maintaining